pkg/ethtypes/hexbytes.go

```diff
@@ -62,6 +62,14 @@ func (h HexBytes0xPrefix) MarshalJSON() ([]byte, error) {
 	return []byte(fmt.Sprintf(`"%s"`, h.String())), nil
 }
 
+func (h HexBytes0xPrefix) Truncate(length int) (HexBytes0xPrefix, error) {
+	if length > len(h) {
+		return nil, fmt.Errorf("truncation length %d larger than length of hex bytes", length)
```

Comment:
It's not a common behaviour that truncates throws an error when the length is smaller than the truncate size.

It makes me think the truncate size is actually convey the constraint of minSize implicitly.

Reply:
Agree - changed the logic here to return the original hex bytes if the length of the bytes is less than the desired length